x-pack/osquerybeat/beater/osquerybeat.go

```diff
@@ -278,7 +278,11 @@ func (bt *osquerybeat) runOsquery(ctx context.Context, b *beat.Beat, osq *osqd.O
 				bt.log.Info("context cancelled, exiting")
 				return ctx.Err()
 			case inputConfigs := <-inputCh:
-				configPlugin.Set(inputConfigs)
+				err = configPlugin.Set(inputConfigs)
+				if err != nil {
+					bt.log.Errorf("failed to set configuration from inputs: %v", err)
+					return err
```

Comment:
actually not sure if we should fail to run osquerybeat, in this case in particular because of bad ecs mapping or maybe we should just log the invalid/rejected mapping and continue running. thoughts?